ci/tasks/deploy-account-management.yml

```diff
@@ -20,7 +20,7 @@ params:
   LOGGING_ENDPOINT_ARN: "arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prod"
   LOGGING_ENDPOINT_ENABLED: "true"
   DEPLOY_LISTENER: "true"
-  SUPPORT_INTERNATIONAL_NUMBERS: "0"
+  SUPPORT_INTERNATIONAL_NUMBERS: "1"
```

Comment:
This file was used whilst we deployed this app via concourse and could be removed now.

Reply:
Hi @JHjava - we have some [tickets ](https://govukverify.atlassian.net/browse/ATB-203)to do this as part of migration that we will do once the migration and renaming is done - until then could we leave it in to be consistent.

Reply:
Ok cool